static/docs/user-guide/contributing-documentation.md

```diff
@@ -105,12 +81,13 @@ follow the steps below:
 - **Markdown and JS files:** content must be properly formatted at 80 symbols
   width. We recommend using Visual Studio Code and the
   [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)
-  plugin to format the content of Markdown files or install the pre-commit hook
-  described above.
+  plugin to format the content of Markdown files or it will be done by a
+  pre-commit hook which was automatically integrated when we ran `npm install`.
```

Comment:
Guys, note that Rewrap and Prettier work differently. Rewrap breaks link titles for example, while Prettier does not.

I think we should end the above sentence at "plugin." (add period) and add a following one with the rest of the text: `Correct formatting will be done automatically by a Git pre-commit hook which was integrated when `npm install` ran.`